packages/neo-one-client/src/types.ts

```diff
@@ -647,7 +654,10 @@ export interface UserAccountProvider {
   readonly selectAccount: (id?: UserAccountID) => Promise<void>;
   readonly deleteAccount: (id: UserAccountID) => Promise<void>;
   readonly updateAccountName: (options: UpdateAccountNameOptions) => Promise<void>;
-
+  readonly executeCompiledCode: (
+    compiledCode: Buffer,
+    options: { readonly from: UserAccountID },
+  ) => Promise<RawInvocationResult>;
```

Comment:
This should return a `TransactionResult<InvokeReceiptInternal>` to be consistent with the equivalent `Client` methods.

Also, rename `InvokeReceiptInternal` to `RawInvokeReceipt` since it's no longer just an internal type.